src/pg/sql/42_observatory_exploration.sql

```diff
@@ -323,7 +323,8 @@ CREATE OR REPLACE FUNCTION cdb_observatory.OBS_GetAvailableGeometries(
   filter_tags TEXT[] DEFAULT NULL,
   numer_id TEXT DEFAULT NULL,
   denom_id TEXT DEFAULT NULL,
-  timespan TEXT DEFAULT NULL
+  timespan TEXT DEFAULT NULL,
+  number_geoms INTEGER DEFAULT NULL -- https://github.com/CartoDB/observatory-extension/blob/1.7.0/src/pg/sql/42_observatory_exploration.sql#L542
```

Comment:
Minor: Do you need the comment now that we don't have 3000 in the code?

Reply:
Removed